pkg/kubectl/cmd/config/create_context_test.go

```diff
@@ -0,0 +1,115 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package config
+
+import (
+	"bytes"
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"k8s.io/client-go/tools/clientcmd"
+	clientcmdapi "k8s.io/client-go/tools/clientcmd/api"
+)
+
+type createContextTest struct {
+	config         clientcmdapi.Config //initiate kubectl config
```

Comment:
Sorry for not being very clear.
I mean we want an additional field describing what this test case is about e.g. "testing creating context"
```
type createContextTest struct {
  field string
  ...
}
```
And we the test is failing, we can have something like:
`"Failed in %q:\nexpected:\n%v\ngot:\n%v\n, test.description, test.expected, actual`
With a description, developer can easily locate which test is failing.